kong/clustering/init.lua

```diff
@@ -53,22 +48,29 @@ function _M:init_cp_worker(plugins_list)
 
   events.init()
 
+  self.wrpc_handler =
+    require("kong.clustering.wrpc_control_plane").new(self.conf, self.cert_digest)
+
   self.wrpc_handler:init_worker(plugins_list)
 end
 
+
 function _M:init_dp_worker(plugins_list)
   local start_dp = function(premature)
     if premature then
       return
     end
 
-    self.child = require("kong.clustering.wrpc_data_plane").new(self.conf, self.cert, self.cert_key)
-    self.child:init_worker(plugins_list)
+    self.wrpc_handler =
+      require("kong.clustering.wrpc_data_plane").new(self.conf, self.cert, self.cert_key)
+
+    self.wrpc_handler:init_worker(plugins_list)
```

Comment:
Same here. Is it necessary for every worker to load its own copy of `wrpc_data_plane`, especially given that only #0 is actually using it?

Reply:
But it's required for every worker. I guess the original idea to require in phase handler is to reduce the `require` cost for non-DP(CP or traditional), but this causes multiple copy.